docs/integrations/containers-orchestration/docker-ulm.md

```diff
@@ -39,7 +39,7 @@ There are alternative methods for collecting Docker logs and metrics. See [Docke
 
 * We recommend running the Sumo collector on the same host where you run Docker. You can deploy the collector as a Docker container. For more information, see [https://hub.docker.com/r/sumologic/collector/](https://hub.docker.com/r/sumologic/collector/).
 * The Sumo Logic Collector uses the Docker Remote API to collect Docker logs. This requires that the log driver configured on the container uses either the `json-file` or `journald` option, as described in [https://docs.docker.com/engine/admin/logging/overview/](https://docs.docker.com/engine/admin/logging/overview/).
-* If you're using Kubernetes we recommend using an open source [Kubernetes Fluentd plugin](/docs/send-data/collect-from-other-data-sources/kubernetes-fluentd-plugin.md).
+* If you're using Kubernetes we recommend using an open source [Sumologic Kubernetes Collection Helm Chart](https://github.com/SumoLogic/sumologic-kubernetes-collection), to learn more see [Kubernetes Observability](/docs/observability/kubernetes).
 * The Docker Log Source uses timestamps from logs to track collection. You need to ensure your log format has a [well-defined timestamp](/docs/send-data/reference-information/time-reference) and the Source is configured to detect it properly. If there are issues with timestamp detection and the Docker container is restarted the Source will reingest all log data since there are no timestamps to track.
```

Comment:
```suggestion
* The Docker Log Source uses timestamps from logs to track collection. You need to ensure your log format has a [well-defined timestamp](/docs/send-data/reference-information/time-reference) and the Source is configured to detect it properly. If there are issues with timestamp detection and the Docker container is restarted, the Source will reingest all log data since there are no timestamps to track.
```

Reply:
Committing minor formatting changes.